src/components/NavBar.vue

```diff
@@ -39,20 +63,25 @@ export default Vue.extend({
 <style scoped lang="scss">
 @import '@/assets/scss/_variables.scss';
 
+$icon-size: 2.5rem;
```

Comment:
I think `$icon-size` would be useful in _variables.scss since we plan on adding more icons to the nav bar

Reply:
I think adding here makes more sense, since as you said, there will be more icons in _nav bar_...